openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIComponentMapper.java

```diff
@@ -776,8 +822,8 @@ private void extractedConstraintAnnotation(MetadataNode metadata,
                                                ConstraintAnnotation.ConstraintAnnotationBuilder constraintBuilder) {
         NodeList<AnnotationNode> annotations = metadata.annotations();
         annotations.stream().filter(annot -> (annot.annotReference() instanceof QualifiedNameReferenceNode &&
-                                ((QualifiedNameReferenceNode) annot.annotReference()).modulePrefix().text()
-                                        .equals("constraint")))
+                        ((QualifiedNameReferenceNode) annot.annotReference()).modulePrefix().text()
```

Comment:
Rather than checking the name of the annotation, we should check the organization name and module name from the annotation symbol right?

Reply:
Currently, implementation of Ballerina to OpenAPI spec is done via utilizing the syntax tree. We're planning a transition to the semantic model for this purpose soon. These improvements will be added through a separate PR.